### PR TITLE
[UXP-22407] - Add support of SWC sp-swatch and sp-swatch-group (v0.37.0)

### PR DESCRIPTION
**Pull request by inasirhussain**

### CODE REVIEW

File: packages/swatch/package.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    "main": "src/index.js",
+    "module": "src/index.js",
+    "type": "module",
+    "exports": {
+        ".": "./src/index.js",
+        "./package.json": "./package.json",
+        "./src/Swatch.js": "./src/Swatch.js",
+        "./src/swatch.css.js": "./src/swatch.css.js",
+        "./sp-swatch.js": "./sp-swatch.js",
+        "./src/SwatchGrouo.js": "./src/SwatchGroup.js",
```
Comment (deepthi-majji) [inline packages/swatch/package.json:24]: Typo: SwatchGroup

File: packages/swatch/src/uxp-swatch.css [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+:host:before {
+    top: 0;
+    left: 0;
+    bottom: 0;
+    right: 0;
+}
+
+.fill:before {
+    box-shadow: -1 -1 0 var(
```
Comment (deepthi-majji) [inline packages/swatch/src/uxp-swatch.css:54]: Can you please tell what does this box-shadow draw?

Comment (inasirhussain) [reply packages/swatch/src/uxp-swatch.css]: This is unused style. Have removed it.

File: packages/swatch/src/uxp-swatch.css [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+/* write uxp style overrides */
+
+:host {
+    --uxp-spectrum-swatch-disabled-icon-color: transparent;
+    --uxp-polyfill-for-clip-path: 4px;
```
[the remark this answers is not in the document]
Comment (inasirhussain) [reply packages/swatch/src/uxp-swatch.css]: Replaced it with related token variable

**Review by deepthi-majji — APPROVED** [review]

**Pull request merged**